src/Core/Asset/VolumeData.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <Core/Asset/AssetData.hpp>
+#include <Core/Types.hpp>
+#include <Core/Utils/Color.hpp>
+
+namespace Ra {
+namespace Core {
+namespace Geometry {
+class AbstractVolume;
+} // namespace Geometry
+
+namespace Asset {
+
+/**
+ * Participating media defined by a grid density
+ */
+struct VolumeData : public AssetData {
+    inline VolumeData( const std::string& name = "" ) : AssetData( name ) {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'VolumeData::volume' is not initialized in the constructor.](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4401329)